R/dash.R

```diff
@@ -487,13 +488,9 @@ Dash <- R6::R6Class(
       dash_reload_hash <- paste0(self$config$routes_pathname_prefix, "_reload-hash")
       route$add_handler("get", dash_reload_hash, function(request, response, keys, ...) {
         modified_files <- private$modified_since_reload
-        hard <- FALSE
+        hard <- TRUE
```

Comment:
@alexcjohnson once we 🔪 `hard` from `dash-renderer`, as outlined in plotly/dash#977, I'll 🔪 it here too.